src/neuroconv/datainterfaces/ophys/inscopix/inscopiximagingdatainterface.py

```diff
@@ -0,0 +1,57 @@
+from typing import Literal
+
+from pydantic import FilePath, validate_call
+
+from ..baseimagingextractorinterface import BaseImagingExtractorInterface
+
+
+class InscopixImagingInterface(BaseImagingExtractorInterface):
+    """Data Interface for Inscopix Imaging Extractor."""
+
+    display_name = "Inscopix Imaging"
+    associated_suffixes = (".isxd",)
+    info = "Interface for handling Inscopix imaging data."
+
+    @validate_call
+    def __init__(
+        self,
+        file_path: FilePath,
+        verbose: bool = False,
+        photon_series_type: Literal["OnePhotonSeries", "TwoPhotonSeries"] = "OnePhotonSeries",
```

Comment:
This is a conversion that will be inhertied from the add_to_nwbfile method. You don't need it at initialization:

https://github.com/catalystneuro/neuroconv/blob/5b6175415f35cbd47f4816d5f702649a25a0e0bb/src/neuroconv/datainterfaces/ophys/baseimagingextractorinterface.py#L140-L152